src/lib/Scenes/Artwork/Components/AboutArtist.tsx

```diff
@@ -0,0 +1,34 @@
+import { Flex, Sans } from "@artsy/palette"
+import { AboutArtist_artwork } from "__generated__/AboutArtist_artwork.graphql"
+import { ArtistListItemContainer as ArtistListItem } from "lib/Components/ArtistListItem"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface AboutArtistProps {
+  artwork: AboutArtist_artwork
+}
+
+export class AboutArtist extends React.Component<AboutArtistProps> {
```

Comment:
@ashkan18 - We should avoid using classes where possible and stick to functional components. (Since the release of hooks classes have been effectively deprecated.)